### PR TITLE
cvat-ui in docker (serve using nginx)

### LINKED ISSUE #488: Dockerize new UI app
We need to dockerize react starter project to run all services simultaneously with `docker-compose up`.

https://github.com/opencv/cvat/pull/464

### PR DESCRIPTION
**Pull request by nmanovic**
Fix #488

### CODE REVIEW

**Comment by nmanovic** [comment]
@azhavoro , please review.

File: .dockerignore [hunk 1]
```diff
@@ -8,3 +8,6 @@
 /keys
 package-lock.json
 node_modules
+cvat-ui
+cvat-core/node_modules
```
Comment (bsekachev) [inline .dockerignore]: Doesn't the line 10 work?

File: cvat-ui/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Install necessary apt packages
+RUN apt update && apt install -yq nodejs npm curl && \
+    npm install -g n && n 10.16.3
+
+# Create output directory
+RUN mkdir /tmp/cvat-ui
+WORKDIR /tmp/cvat-ui/
+
+# Install dependencies
+COPY package.json /tmp/cvat-ui/
```
Comment (bsekachev) [inline cvat-ui/Dockerfile]: package.json && package-lock.json

File: .dockerignore (hunk 1 above)
Comment (nmanovic) [reply .dockerignore]: It seems it doesn't. Better to have '**/node_modules' pattern here.

File: cvat-ui/Dockerfile (hunk 2 above)
Comment (nmanovic) [reply cvat-ui/Dockerfile]: OK

**Pull request merged**